src/tox/helper/build_isolated.py

```diff
@@ -0,0 +1,14 @@
+import sys
+
+backend_spec = sys.argv[1]
```

Comment:
wrap each of these scripts in `if __name__ == '__main__':` just for sanity sake?

maybe also give them `dashed-names.py` to better indicate that they aren't importable modules?

Maybe also name this directory `scripts` instead of `helpers`

(all minor suggestions, feel free to take and/or leave any of them!)